src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ScrollViewer.cs

```diff
@@ -1244,7 +1245,10 @@ public async Task When_TouchScroll_Then_NestedElementReceivePointerEvents()
 			var sutLocation = sut.GetAbsoluteBounds().GetLocation();
 			finger.Drag(sutLocation.Offset(5, 480), sutLocation.Offset(5, 5));
 
-			events.Should().BeEquivalentTo("enter", "pressed", "release", "exited");
+			// The expected proper sequence should really be "Enter", "Pressed", "PointerCaptureLost", "Exited"
+			// This will need proper equivalent of Windows' Direct Manipulation.
+			// For now, it's important to not get Released as it can cause a click on the nested element while it's being scrolled.
+			events.Should().BeEquivalentTo("enter", "pressed", "exited");
```

Comment:
@dr1rrb It's very unfortunate that we'll get pressed, without one of released/capture lost/cancel. I assume it's okay for now, but I'm thinking we may really want to match WinUI in future.

Reply:
It's 100% normal for an element to get a pointer pressed without any released / capture lost / cancel. On UWP/WinUI the "pressed" and "release" are raised on the elements that are under the pointer when the event occurs (unless you capture it). There is no notion of "sequence" -what I call "implicit capture"- unlike on iOS and Android.